.github/workflows/ci.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [8, 10, 12, 13]
+        node: [10, 12, 13]
```

Comment:
We can't quite do this yet, right? This is still technically the legacy-8 branch (we haven't shifted to the 10+ deprecation on this repo).

Reply:
Generator overwrite this.
I could add hack in synth.py

Reply:
Taking back what I said after the conversation about the breaking changes - maybe we should try to roll this and the Node 8 deprecation into one major release.

Reply:
try to isolate conversion logic from Node 8 deprecation.
I will add one more hack in synth.py to add Node 8 in ci.yaml